src/annotator/integrations/pdf.js

```diff
@@ -270,10 +274,10 @@ export class PDFIntegration {
 
     render(
       <Banners>
-        {this._bannerState.contentPartner && (
-          <ContentPartnerBanner
-            provider={this._bannerState.contentPartner}
-            onClose={() => this._updateBannerState({ contentPartner: null })}
+        {this._bannerState.contentInfo && (
+          <ContentInfoBanner
+            info={this._bannerState.contentInfo}
+            onClose={() => this._updateBannerState({ contentInfo: null })}
```

Comment:
It's still up in the air whether we'll have a close button or not, but it's easy enough to muddle with that when the time comes.